src/nccl_ofi_rdma.c

```diff
@@ -2776,6 +2885,24 @@ static inline int reg_mr_on_device(nccl_net_ofi_rdma_ep_t *ep,
 		}
 	}
 
+	/* Register memory on each control rail */
```

Comment:
Once we split control and eager message buffers, we should avoid this double registration.  Can you open a SIM to only allocate on the domains needed?

Reply:
Done